cmd/ddev/cmd/cd.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/ddev/ddev/pkg/ddevapp"
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/output"
+	"github.com/ddev/ddev/pkg/util"
+	"github.com/spf13/cobra"
+	"path/filepath"
+)
+
+var bashFile = filepath.Join(globalconfig.GetGlobalDdevDir(), "commands/host/shells/ddev.sh")
+var fishFile = filepath.Join(globalconfig.GetGlobalDdevDir(), "commands/host/shells/ddev.fish")
+
+// CdCmd is the top-level "ddev cd" command
+var CdCmd = &cobra.Command{
+	Use:   "cd [project-name]",
+	Short: "Uses shell built-in 'cd' to change to a project directory",
+	Long: fmt.Sprintf(`To enable the 'ddev cd' command, source the ddev.sh script from your rc-script.
```

Comment:
Is this needed? I didn't need it... I love this.

Reply:
It's needed, I just tested my PR on a test runner, and it prints the directory:

```
$ ddev cd TestPkgDrupal10
/Users/testbot/tmp/ddevtest/TestPkgDrupal10581214420
```

It doesn't change the directory.

Please check if you already have some customizations.

Reply:
Same for me, on both bash and zsh. Without the shell function, it just prints the directory.